client/src/app/modules/model-manager/components/omz-import-ribbon-content/omz-import-ribbon-content.component.html

```diff
@@ -16,25 +13,33 @@
 
     <ng-container ngProjectAs="wb-model-zoo-filter-group" [ngTemplateOutlet]="filterGroupTemplate"></ng-container>
 
-    <wb-search-field
+    <wb-config-form-field
       ngProjectAs="model-zoo-search"
-      (search)="searchModels($event)"
+      [field]="searchField"
+      [group]="sortAndSearchFormGroup"
       testId="search-field"
-    ></wb-search-field>
+    ></wb-config-form-field>
 
     <wb-model-zoo-counter
+      *ngIf="dataSource?.data.length"
       [dataLength]="dataSource?.data.length"
       [filteredDataLength]="dataSource?.filteredData?.length"
     ></wb-model-zoo-counter>
 
     <mat-paginator
+      #matPaginator
       ngProjectAs="model-zoo-paginator"
       [length]="dataSource.data.length"
       [pageSize]="8"
       hidePageSize
     ></mat-paginator>
 
-    <wb-card-grid [dataSource]="dataSource">
+    <wb-card-grid
+      [dataSource]="dataSource"
+      [isLoading]="isLoading"
+      *ngIf="dataSource?.filteredData?.length || isLoading; else emptyModelsTemplate"
+    >
+      <ng-container *wbSkeleton="isLoading; repeat: matPaginator.pageSize; height: '100px'"></ng-container>
```

Comment:
Maybe such structural directive for skeleton loader is too complicated. There is an option to create separate components, e.g. for for card skeleton loader that will include just `wb-rectangle` components. And use such static component here with `*ngIf` here instead of `ng-container`